tests/org.eclipse.passage.loc.billing.core.tests/src/org/eclipse/passage/loc/billing/core/tests/StringsProvider.java

```diff
@@ -0,0 +1,37 @@
+package org.eclipse.passage.loc.billing.core.tests;
+
+import java.nio.charset.Charset;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class StringsProvider {
+
+	private final List<String> strings = new LinkedList<String>();
+	private final Random random = new Random();
+
+	private boolean contains(String string) {
+		return strings.contains(string);
+	}
+
+	private boolean add(String string) {
+		return strings.add(string);
+	}
+
+	public final void clear() {
+		strings.clear();
+	}
+
+	public final String randomString() {
+		byte[] array = new byte[5];
+		random.nextBytes(array);
+
```

Comment:
An [empty line](https://www.yegor256.com/2014/11/03/empty-line-code-smell.html) in a method body most probably means there are actually two methods.  Take a  closer look at the design. You use `array` just ones, can it be a direct call for a private helper? Like `new String(array(), Charset...`.